kepler/parsers.py

```diff
@@ -120,3 +124,12 @@ def convert_coord(cls, coordinate, precision=10):
             dec = dec * -1
         getcontext().prec = o_precision
         return dec
+
+    @classmethod
+    def pad_034(self, coordinate):
+        c = list(coordinate)
+        if c[0] in ['N', 'E', 'W', 'S'] and len(c) == 7:
```

Comment:
Thanks. I had seen mention of that in my googling but hadn't quite grokked the syntax yet.